docker/check-status.sh

```diff
@@ -1,8 +1,14 @@
-. /tmp/iip-configuration.sh
+#!/bin/bash
 
-COUNT=$(ps aux | grep /opt/iipsrv/src/iipsrv.fcgi | grep -v grep | wc -l)
-if [ $COUNT -lt $NB_IIP_PROCESS ]; then
-    echo "$COUNT / $NB_IIP_PROCESS on $(date)"
-    killall -9 /opt/iipsrv/src/iipsrv.fcgi
-    /tmp/start-iip.sh
-fi;
+while true; do
+    COUNT=$(ps aux | grep /tmp/start-iip-proc.sh | grep -v grep | wc -l)
+        if [ $COUNT -lt $NB_IIP_PROCESS ]; then
+            echo "$COUNT / $NB_IIP_PROCESS on $(date)"
+            killall -9 /tmp/start-iip-proc.sh
+            killall -9 /opt/iipsrv/src/iipsrv.fcgi
```

Comment:
If for some reason this check-status.sh script is kept in use, contrary to my thinking in my comment on above line in this file under [this thread](https://github.com/Proscia/iipsrv/pull/9/files#r1657787305), shouldn't both of these lines be removed now, or else it would continue killing all of the processes if any 1 process dies?

Reply:
No - when the iipsrv processes themselves die, they automatically respawn via start-iip-proc.sh, if there are missing start-iip-proc processes, then something other than a simple request-failed-iipsrv-proc crashed situation has happened, so it makes sense to just get back into a "normal" state. 

The alternative would be that we have to figure out which start-iip-proc is "missing" based on the PORT and make sure to kill its associated iipsrv.fcgi proc if it is running.

As this is all a less likely scenario now, an infrequent "additional" status check to get things back to a normal state seems ok.

Reply:
👍 I think I'm following your train of thought now, assuming we expect the start-*.sh processes to somehow die.. but I'm still skeptical of why/when that would actually happen and this feels like YAGNI logic.